### PR TITLE
Fix #1038 add TLDs

### LINKED ISSUE #1038: Add more TLDs
### Prerequisites

- [X] I checked the documentation and understood it;
- [X] I checked to make sure that this issue has not already been filed;

### Problem description

Some TLDs not included in filter lists

### Proposed solution

Add more TLDs

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/FiltersRegistry/issues/1038

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: scripts/wildcard-domain-processor/top-tld.ts [hunk 1]
```diff
@@ -95,15 +97,6 @@ export const TOP_LEVEL_DOMAIN_LIST = [
...
     'gob.mx',
-    'gov',
-    'gov.bd',
-    'gov.co',
-    'gov.in',
-    'gov.my',
-    'gov.np',
-    'gov.tr',
-    'gov.ua',
-    'gov.vn',
```
Comment (AdamWr) [inline scripts/wildcard-domain-processor/top-tld.ts:106]: Any reason why these TLDs have been removed?

Comment (jellizaveta) [reply scripts/wildcard-domain-processor/top-tld.ts:106]: They count as registered domain and added to almost all wildcards

Comment (jellizaveta) [reply scripts/wildcard-domain-processor/top-tld.ts:106]: @Alex-302 maybe leave just `gov` ?

**Review by AdamWr — APPROVED** [review]

File: scripts/wildcard-domain-processor/top-tld.ts (hunk 1 above)
Comment (Alex-302) [reply scripts/wildcard-domain-processor/top-tld.ts:106]: Can't remember examples, where `gov` domains are required.

**Review by zloyden — APPROVED** [review]

**Pull request merged**